CMakeLists.txt

```diff
@@ -451,29 +448,51 @@ else()
     util/io_posix.cc)
 endif()
 
+if(WIN32)
+  set(SYSTEM_LIBS ${SYSTEM_LIBS} Shlwapi.lib Rpcrt4.lib)
+  set(ROCKSDB_STATIC_LIB rocksdblib${ARTIFACT_SUFFIX})
+  set(ROCKSDB_SHARED_LIB ${ROCKSDB_STATIC_LIB})
```

Comment:
It's possible that I have the import vs static inverted. Please comment if that's the case.